gensim/models/base_any2vec.py

```diff
@@ -0,0 +1,792 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from gensim import utils
+import logging
+from timeit import default_timer
+import threading
+import numpy as np
+from six import string_types
+from six.moves import xrange
+from numpy import vstack
+from gensim import matutils
+from numpy import float32 as REAL, ones, random, argmax
+from types import GeneratorType
+from gensim.utils import deprecated
+
+try:
+    from queue import Queue
+except ImportError:
+    from Queue import Queue
+
+logger = logging.getLogger(__name__)
+
+
+# Public Interfaces
```

Comment:
It's better to add this tip to class docstring (instead of comment in code)

Reply:
Sure, let me complete the doc string for `base_any2vec.py`.